apps/transmitmedia.cpp

```diff
@@ -1005,13 +1005,17 @@ class UdpCommon
 
             if (attr.count("source"))
             {
+#ifdef IP_ADD_SOURCE_MEMBERSHIP
```

Comment:
Please configure with `--enable-testing` and recompile (`-DENABLE_TESTING=1` if you use cmake directly). Likely the same fix is needed in `testing/testmedia.cpp` file.

Reply:
> Likely the same fix is needed in `testing/testmedia.cpp` file.

? No, that file doesn't use `IP_ADD_SOURCE_MEMBERSHIP`.

Reply:
Oh... you're right. I was stating they are perfectly synchronized. Something to do in the future.